### PR TITLE
fix(discount): scrape transactions for all user accounts

### LINKED ISSUE #1011: Support multi account scraping for Bank Discount
Currently the scraper returns only the default account

### PR DESCRIPTION
**Pull request by Arielgordon123**
closes #1011

### CODE REVIEW

**Comment by baruchiro** [comment]
Hi, next time tag me or send me a whatsapp

File: src/scrapers/discount.ts [hunk 1]
```diff
@@ -73,42 +78,47 @@ async function fetchAccountData(page: Page, options: ScraperOptions): Promise<Sc
       errorMessage: 'failed to get account data',
     };
   }
-  const accountNumber = accountInfo.UserAccountsData.DefaultAccountNumber;
 
   const defaultStartMoment = moment().subtract(1, 'years').add(2, 'day');
```
Comment (baruchiro) [inline src/scrapers/discount.ts:82]: Interesting, why's the previous developer used the `DefaultAccountNumber`, if the response includes all acounts?

I'm trying to say, we need someone with only one account to test your change, maybe for one account only, the `UserAccounts` is not exists or empty?

Comment (yyogo) [reply src/scrapers/discount.ts:82]: Hi, just tested, works with one account
